README.md

```diff
@@ -164,7 +164,7 @@ Now you'll need to do steps (2) everytime you make a changes and want to build t
 
 **Right now, this is only tentative guide. Errata may exist. Please report errors on the [gitter channel](https://gitter.im/openwisp/dockerize-openwisp?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge).**
 
-The Makefile has following options, that are useful for developers & users. Default action is `compose-build`.
+The Makefile has following options, that are useful for developers & users. Default action is `compose-build`. For building images, you can use different python package sources by changing `.makerc`.
```

Comment:
I placed the `.makerc` notice here instead of in build-base because almost every make command (except `make clean`, obviously) build the base image, so it's should be a general knowledge.

Reply:
No, I think this point is very importatnt.
I would add it to "Build (Development)" section.

```
### Building with your python packages

You can build with your own package....(all the steps)...
```

Can you do something like that?